sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsBufferCatalog.scala

```diff
@@ -298,4 +299,9 @@ object RapidsBufferCatalog extends Logging with Arm {
 
   /** Remove a buffer ID from the catalog and release the resources of the registered buffer. */
   def removeBuffer(id: RapidsBufferId): Unit = singleton.removeBuffer(id)
+
+  /** Get diskBlockManager. */
+  def getDiskBlockManager(): RapidsDiskBlockManager = {
+    diskBlockManager
+  }
```

Comment:
Lets remove the parenthesis (since this method is just an accessor) and we can also just return `diskBlockManager` in a single line:

```suggestion
  def getDiskBlockManager: RapidsDiskBlockManager = diskBlockManager
```